tox.ini

```diff
@@ -80,6 +80,17 @@ deps =
 [testenv:provider]
 extras = provider
 
+[testenv:pinned-provider]
+# zyte-common-items ≥ 0.6.0 requires Python ≥ 3.8.
+basepython=python3.8
+extras = provider
+deps =
+    # scrapy-poet 0.10.0 depends on scrapy>=2.6.0
+    {[testenv:pinned-scrapy-2x6]deps}
+    scrapy-poet==0.10.0
+    web-poet==0.13.0
+    zyte-common-items==0.7.0
```

Comment:
Could you please add a comment to setup.py, to ensure we don't forget to update tox.ini when the minimum dependencies change?